irc/hostserv.go

```diff
@@ -415,9 +420,8 @@ func hsTakeHandler(server *Server, client *Client, command string, params []stri
 		} else {
 			hsNotice(rb, client.t("An error occurred"))
 		}
-	} else if vhost != "" {
-		hsNotice(rb, client.t("Successfully set vhost"))
 	} else {
-		hsNotice(rb, client.t("Successfully cleared vhost"))
+		hsNotice(rb, client.t("Successfully set vhost"))
+		server.snomasks.Send(sno.LocalVhosts, fmt.Sprintf("Client %s (account %s) took vhost %s", client.Nick(), account, vhost))
```

Comment:
this line brings up a question for me: should we translate snomask lines to the default server language? if so we'll want to make this use the `%[1]s` `%[2]s` stuff instead